homeworks/becseipeter/5_oop/car_fleet_mgr.py

```diff
@@ -0,0 +1,79 @@
+class Car:
+    
+    def __init__(self, brand: str, model: str, year: int):
+        self.brand = brand
+        self.model = model
+        self.year = year
+        self.mileage = 0                    
+        self.fuel_level = 100               
+
+    def drive(self, kilometers):
+        fuel_needed = kilometers * .1
+        if self.fuel_level > fuel_needed:
+            self.fuel_level -= fuel_needed
+            self.mileage += kilometers
+        else:
+            print(f'Find a gas station.')
+
+    def refuel(self, amount):
```

Comment:
itt is hasonló a helyzet, ellenőrizd kérlek már a metódus elején hogy nem e negatív az amount.